.github/workflows/build-swift.yml

```diff
@@ -0,0 +1,102 @@
+---
+name: Build Swift Package
+
+on:
+  push:
+    branches:
+      - "rc"
+      - "hotfix-rc"
+      - "main"
+
+jobs:
+  version:
+    name: Get Version
+    runs-on: ubuntu-22.04
+    outputs:
+      package_version: ${{ steps.retrieve-version.outputs.package_version }}
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4.0.0
+
+      - name: Get Package Version
+        id: retrieve-version
+        run: |
+          VERSION=$(grep -o '^version = ".*"' crates/bws/Cargo.toml | grep -Eo "[0-9]+\.[0-9]+\.[0-9]+")
```

Comment:
This should be the bitwarden crate not the cli.

Reply:
Whoops.... Fixed 😃

Reply:
@michalchecinski it occured to me know that the bitwarden verison is inherited from the workspace version. https://github.com/bitwarden/sdk/blob/main/Cargo.toml#L8

Reply:
Got it. Also, should the version for SWIFT SDK be the same as Rust crates?

Reply:
I think we can use that for now.